src/pages/StorjDcs.tsx

```diff
@@ -1,15 +1,17 @@
 import React from "react";
-import { useLocation } from "react-router";
+import { useParams } from "react-router";
+import { BucketListProfile } from "../components/BucketListProfile";
+import Head from "../components/Head";
 
-export const StorjDcs = () => {
-  const location = useLocation();
+export const BucketPage = () => {
```

Comment:
I'm not sure we want to change the name of the component, since this is for the Storj DCS profile type. We could possibly have other profile types.